Mage/src/main/java/mage/game/command/emblems/ObNixilisOfTheBlackOathEmblem.java

```diff
@@ -28,9 +27,8 @@ public ObNixilisOfTheBlackOathEmblem() {
         effect.setText("You gain X life");
```

Comment:
another potential enum option something like X_HIDDEN, for cases like this where X will be explained in a subsequent effect